packages/localize-tools/package.json

```diff
@@ -19,13 +19,104 @@
   "main": "lit-localize.js",
   "type": "module",
   "scripts": {
-    "clean": "git clean -dXn | sed 's/^Would remove //' | grep -v node_modules | xargs rm -rf",
-    "build": "npm run clean && tsc --build && npm run generate-json-schema",
-    "generate-json-schema": "typescript-json-schema tsconfig.schema.json ConfigFile --include=lib/types/config.d.ts --required --noExtraProps > config.schema.json",
-    "test": "npm run test:unit && npm run test:check-tsc",
-    "test:unit": "uvu lib/tests '\\.test\\.js$' --ignore 'ssr\\.unit\\.test\\.js$' && uvu lib/tests '\\ssr\\.unit\\.test\\.js$'",
-    "test:update-goldens": "UPDATE_TEST_GOLDENS=true npm run test:unit",
-    "test:check-tsc": "ls testdata/*/output/tsconfig.json | xargs -n 1 tsc --noEmit --project"
+    "build": "wireit",
+    "build:ts": "wireit",
+    "generate-json-schema": "wireit",
+    "test": "wireit",
+    "test:unit": "wireit",
+    "test:unit:ssr": "wireit",
+    "test:update-goldens": "wireit",
+    "test:check-tsc": "wireit"
+  },
+  "wireit": {
+    "build": {
+      "dependencies": [
+        "build:ts",
+        "generate-json-schema",
+        "../lit:build",
+        "../labs/ssr-client:build"
```

Comment:
is this needed for tests?

Reply:
also, the tsconfig reference was to ssr, but this is to ssr-client. Was ssr-client needed but just built transitively?

Reply:
Changed to `ssr:build`. It worked this way because SSR doesn't have a rollup step, so `ssr:build` is just `ssr:build:ts` plus `../ssr-client:build`. But we already had `ssr:build:ts` in the `build:ts` deps here, so the missing piece was `../ssr-client:build`. But it makes more sense to just depend on `../ssr:build`.